docs/mobile-apps/automated-testing/appium/appium-flutter-integration-driver.md

```diff
@@ -0,0 +1,678 @@
+   ```
+   
+   The app will be located in `{project-root}/build/ios/iphonesimulator/`
+
+   For Real Device - Release mode
+   ```bash
+    flutter build ipa --release integration_test/appium_test.dart
+    ```
+   
+   The IPA file will be located in `{project-root}/build/ios/ipa/`
+
+Bingo! You are ready to run your tests using Appium Flutter Integration Driver.
+
+## Uploading your Flutter App to Sauce Labs
+
+You can now upload the built apps with our [REST API](/dev/api/storage/#upload-file-to-app-storage), or manually upload them to the preferred Data Center. See [Manually Uploading an App](/mobile-apps/live-testing/live-mobile-app-testing/#uploading-an-app) for more information.
+
+## Configuring your Appium Capabilities
+
+More information on how to write Appium tests for Flutter apps can be found in the [appium-flutter-integration-driver](https://github.com/AppiumTestDistribution/appium-flutter-integration-driver) repository. You can also find a sample Flutter app and tests in the [Demo JS - Appium Flutter Integration](https://github.com/saucelabs-training/demo-js/tree/main/webdriverio/appium-app/examples/appium-flutter-integration)-repository.
+
+:::info W3C Capabilities
+We encourage you to use W3C capabilities for your tests. For more information on W3C capabilities, see [W3C Capabilities](/dev/test-configuration-options/#mobile-appium-capabilities).
+:::
+
+:::info APPIUM FLUTTER INTEGRATION DRIVER SUPPORT :
+[appium-flutter-integration-driver](https://github.com/AppiumTestDistribution/appium-flutter-integration-driver). This driver will now be included by default with Appium version latest and all subsequent versions released after July 1st.
+
+More information regarding the available [appium versions](/mobile-apps/automated-testing/appium/appium-versions/#Real-Devices) we support at Sauce Sabs.
+:::
+
+### Android Emulators and iOS Simulators
+
+<Tabs
+groupId="capability-ex-emusim"
+defaultValue="java"
+values={[
+{label: 'Java', value: 'java'},
+{label: 'Node.js', value: 'js'},
+{label: 'Python', value: 'python'},
+{label: 'Ruby', value: 'ruby'},
+{label: 'C#', value: 'csharp'},
+]}>
+
+<TabItem value="java">
+<Tabs
+groupId="capability-java-emusim"
+defaultValue="android"
+values={[
+{label: 'Android', value: 'android'},
+{label: 'iOS', value: 'ios'},
+]}>
+<TabItem value="android">
+
+<!-- prettier-ignore -->
+```java
+MutableCapabilities capabilities = new MutableCapabilities();
+
+capabilities.setCapability("platformName", "android");
+capabilities.setCapability("appium:platformVersion", "12");
+capabilities.setCapability("appium:deviceName", "Google Pixel Emulator");
+// Mandatory for using the appium-flutter-integration-driver
+capabilities.setCapability("appium:automationName", "FlutterIntegration");
+capabilities.setCapability("appium:app", "storage:filename=sl_my_demo_flutter_app.apk");
+
+HashMap<String, Object> sauceOptions = new HashMap<String, Object>();
+sauceOptions.put("appiumVersion", "appium2-20240701");
```

Comment:
What about `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` inside `sauceOptions`?

Reply:
done